atc/configvalidate/validate.go

```diff
@@ -254,6 +254,11 @@ func validateJobs(c Config) ([]ConfigWarning, error) {
 
 	names := map[string]int{}
 
+	if len(c.Jobs) == 0 {
+		errorMessages = append(errorMessages, "no jobs found. pipeline must contain at least one job")
```

Comment:
@taylorsilva Non-blocking nitpick: Do you think this could be simplified and made to be more consistent with the other messages? This is a perhaps a bit pedantic -- so feel free to ignore me -- but it makes me a bit twitchy that this is two sentences, but only the first has a period and neither is capitalized.

```suggestion
		errorMessages = append(errorMessages, "pipeline must contain at least one job")
```

Reply:
I can drop the `no jobs found` part. It was something I added after writing the initial test + implementation.
I'm going to leave everything lowercase because it's idiomatic for go for error messages to not start capitalized.

Reply:
Sounds good; that makes sense. Thanks for this!